src/utils/src/arg_parser.rs

```diff
@@ -150,20 +186,40 @@ impl<'a> Argument<'a> {
         self
     }
 
-    fn format_help(&self) -> String {
+    fn format_help(&self, max_arg_width: usize) -> String {
```

Comment:
Hmm, for this function context, I don't think `max_arg_width` is an accurate name, it's more of a fixed width than max. I think I prefer the old version (`arg_width` if I remember correctly).

Reply:
Done. Thank you for the comment.